src/sentry/models/grouphashmetadata.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, JSONField, Model, region_silo_model
+
+
+@region_silo_model
+class GroupHashMetadata(Model):
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
I do not understand the implications of using this.

Reply:
This is part of that backup/relocation project Alex has been working on. When you migrate an org from one region to another, or from on-prem to SAAS, not all of your data comes along for the ride, and in particular none of your event/group data does.